ObjectiveMe/ObjectiveMe/Person.h

```diff
@@ -0,0 +1,22 @@
+//
+//  Person.h
+//  ObjectiveMe
+//
+//  Created by Rachael Alees McKinney on 2/4/16.
+//  Copyright © 2016 CodeLab. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface Person : NSObject
+
+
+@property NSString *firstname;
+@property NSString *lastname;
+@property NSString *age;
```

Comment:
I would do NSNumber, NSInteger or even a plain int here. The reason is that age is always going to be some number of years – an integer, in our case. To illustrate the benefit to using a numerical data type here, consider a case where you wanted to sort out all Person objects that were over age 25. If you are using numerical data, you can use operators such as < ,> and == to work with your data.